code-of-conduct.md

```diff
@@ -0,0 +1,41 @@
+## CoreOS Community Code of Conduct
+
+### Contributor Code of Conduct
+
+As contributors and maintainers of this project, and in the interest of
+fostering an open and welcoming community, we pledge to respect all people who
+contribute through reporting issues, posting feature requests, updating
+documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
+experience for everyone, regardless of level of experience, gender, gender
+identity and expression, sexual orientation, disability, personal appearance,
+body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing others' private information, such as physical or electronic addresses, without explicit permission
+* Other unethical or unprofessional conduct.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct. By adopting this Code of Conduct,
+project maintainers commit themselves to fairly and consistently applying these
+principles to every aspect of managing this project. Project maintainers who do
+not follow or enforce the Code of Conduct may be permanently removed from the
+project team.
+
+This code of conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting a project maintainer or Brandon Philips
+<brandon.philips@coreos.com>.
```

Comment:
Should we have a different contact point?

Reply:
Yes, I don't know what it should be. Probably a list of maintainers at this point. Ideally someone who has had some sort of Code of Conduct or at the very least read a book on the topic like this: https://frameshiftconsulting.com/code-of-conduct-book/

Reply:
That makes sense. Thanks @philips. 

@ajeddeloh @arithx Why don't we leave it at `contacting a project maintainer from MAINTAINERS` for now.

Reply:
SGTM. Added a MAINTAINERS file and dropped brandon.